### ISSUE TITLE
Expand  Error validating PayPal credentials. Unauthorized to be more specific on how to get back on track

### ISSUE THREAD

**Issue by lorenzocaum** [eventespresso/eea-paypal-smart-buttons#27]
**Summary**
PayPal Smart Buttons is doing well on Event Smart. Most event organizers follow the steps in the help doc and get set up.

We've had a couple of reports where they've received 401 not authorized which I know is bad credentials.

I attempted to reproduce that and I see that this is shown:

Error validating PayPal credentials. Unauthorized

![image 2018-11-02 at 7 40 26 am](https://user-images.githubusercontent.com/1079138/47913464-96a78880-de72-11e8-8ce9-06ef5b56cf05.png)

Could we adjust that so it includes feedback on how to get back on track?

Here are some suggestions:

- Your API credentials are not valid. Double-check that you are copying and pasting the credentials into the correct fields or try following the setup steps again to create a new app.

- PayPal did not accept your API credentials. Double-check your credentials and copy and paste them into the fields again and save changes. Alternatively, create a new app and try the new API credentials.

**Comment by sethshoultes** [comment]
@mnelson4 so are you saying that we should move this ticket to the core plugin, then?

**Comment by mnelson4** [comment]
Yeah I think there should be a ticket in EE core. This can stay if we want to track it's progress in ES. I can do that once I get started on this, or someone else can if they want earlier.

**Comment by lorenzocaum** [comment]
This issue was specific to PayPal Smart Buttons but I think from your feedback, a change is needed in core to handle this so I've created an issue on core: https://github.com/eventespresso/event-espresso-core/issues/772

**Comment by mnelson4** [comment]
> I think we should make the adjustments to EE core though, have everybody benefit.

Errrmm oups. I thought this was in the EventSmart repo- it's actually in the PayPal Smart Buttons repo. 

Nevertheless, this issue probably also applies to the other PayPal gateways...
